src/sdk/namespace_fs.js

```diff
@@ -59,6 +59,41 @@ function isDirectory(ent) {
     }
 }
 
+/**
+ * 
+ * @param {*} stat - entity stat yo check
+ * @param {*} fs_account_config - account config using to check symbolic links
+ * @param {*} entry_path - path of symbolic link
+ * @returns 
+ */
+async function isDirectoryOrSymLinkToDirectory(stat, fs_account_config, entry_path) {
+    try {
+        let r = isDirectory(stat);
+        if (!r && isSymbolicLink(stat)) {
+            let targetStat = await nb_native().fs.stat(fs_account_config, entry_path);
```

Comment:
I'm not sure I understand the logic here. why is that the targetStat? don't you need to check `realpath`?

Reply:
No, if the stat is a symbolic link, then the method tries to get a stat of the target file and check if the target file is a directory. fs.stat() returns a stat of the lowest target, even if there are a few symbolic links on the way (symlink points to symlink and so on)